editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -3419,11 +3419,7 @@ void Node3DEditorViewport::reset() {
 	last_message = "";
 	name = "";
 
-	cursor.x_rot = 0.5;
-	cursor.y_rot = 0.5;
-	cursor.distance = 4;
-	cursor.region_select = false;
-	cursor.pos = Vector3();
+	cursor = Cursor();
```

Comment:
Isn't it default initialized already when declared? This might be redundant.

Reply:
How so? This is in a method to reset it.

Reply:
Ah I misread, I thought this was the constructor.